examples/docker-compose.yml

```diff
@@ -1,14 +1,24 @@
 version: '2'
 services:
   conjur:
-    image: registry.tld/conjur-appliance-cuke-master:4.9-stable
+    image: cyberark/conjur:0.1.0-stable
```

Comment:
Do we have simply a "stable" label? If we push new versions we would like Puppet to test itself against the latest stable, no?

Reply:
Agreed. We don't, however.

Reply:
The `latest` label is our stable label for `cyberark/conjur` - we should use that. Only green builds on master push to `latest`.

Reply:
so, this should just be `image: cyberark/conjur`